clusterloader2/pkg/prometheus/manifests/0ssd-storage-class.yaml

```diff
@@ -7,7 +7,18 @@ metadata:
   name: ssd
 provisioner: {{$PROMETHEUS_STORAGE_CLASS_PROVISIONER}}
 parameters:
+  {{if eq $PROMETHEUS_STORAGE_CLASS_PROVISIONER "kubernetes.io/gce-pd"}}
```

Comment:
It's not directly related to image bump. Please open separate PR (which makes potential rollback easier).